src/clp_logging/handlers.py

```diff
@@ -68,6 +68,21 @@ def _init_timeinfo(fmt: Optional[str], tz: Optional[str]) -> Tuple[str, str]:
     return fmt, tz
 
 
+def _encode_log_event(msg: str, ref_timestamp_ms: int) -> Tuple[bytearray, int]:
+    """
+    Encodes the log event with the input log message and reference timestamp.
+
+    :param msg: Input log message.
+    :param ref_timestamp_ms: Input reference timestamp.
```

Comment:
Lets call it  the last or previous timestamp (try to be consistent with the other code, or make other code consistent with this). Iirc we only call the preamble timestamp the reference timestamp.